Gifski/Utilities.swift

```diff
@@ -608,22 +598,87 @@ extension AVAsset {
 
 	This can be useful to trim blank frames from files produced by tools like the iOS simulator screen recorder.
 	*/
-	func trimBlankFrames() throws -> AVAsset {
+	func trimmingBlankFrames() throws -> AVAsset {
 		guard let videoTrack = self.firstVideoTrack else {
-			throw VideoTrimmingError.assetIsMissingTrack
+			throw AVAssetTrack.VideoTrimmingError.assetIsMissingTrack
 		}
 
-		let trimmedTrack = try videoTrack.trimBlankFrames()
+		let trimmedTrack = try videoTrack.trimmingBlankFrames()
 
 		guard let trimmedAsset = trimmedTrack.asset else {
-			throw VideoTrimmingError.assetIsNil
+			throw AVAssetTrack.VideoTrimmingError.assetIsNil
```

Comment:
You can add a typealias for the error to `AVAsset`.

Reply:
Added in 3a2c8a6.